city_scrapers/spiders/bisnd_bps.py

```diff
@@ -0,0 +1,158 @@
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        selected_title = item[3]
+        html_title_str = lhtml.fromstring(selected_title)
+        title = html_title_str.text_content().strip()
+        return title if title is not None else ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        title = "".join(item[3].split(">")[1].split("<")[:-1])
+        return COMMITTEE if "committee" in title.lower() else BOARD
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        extracted_date = [
+            html.unescape("".join(element.split(">")[1].split("<")[:-1]))
+            for element in item[:3]
+        ]
+        result = " ".join(extracted_date)
+        parsed_date = result if "(" not in result else result.split(" (")[0]
+        date_obj = datetime.strptime(parsed_date, "%Y %B %d")
+
+        return datetime.combine(date_obj, self.meeting_time)
+
+    def _parse_time_notes(self, item):
+        """
+        Meeting detail for non 'Regular' type meetings
+        are accessable from the meeting agenda.
+        The notes are added to further clarify the meeting details.
+        """
+        return "Meetings that are not of type 'Regular' are held at specific locations with specific timing, please refer to the meeting agenda for more details."  # noqa
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        extracted_links = [link for link in item[3:]]
+        parsed_links = []
+        for link in extracted_links:
+            sel = Selector(text=link if link is not None else "")
+            href = sel.css("a::attr(href)").get()
+            title = sel.css("a::text").get()
+            if href is not None and title is not None:
+                # Remove extra spaces from title
+                title = " ".join(title.split())
+                parsed_links.append({"href": href, "title": title})
+            else:
+                continue
+        return parsed_links
+
+    def _get_status(self, item):
+        """Parse or generate status from item."""
+        if "cancel" in item["title"].lower():
+            return CANCELLED
+        if item["start"] < datetime.now():
+            return PASSED
+        return TENTATIVE
```

Comment:
**suggestion:** My apologies, @msrezaie, this probably could be clearer in our documentation. `_get_status` is actually [declared](https://github.com/City-Bureau/city-scrapers-core/blob/44ed57f1f4833f7c35276fff4bc5187e191da496/city_scrapers_core/spiders/spider.py#L72-L84) in the `CityScrapersSpider` class in `city-scrapers-core` so we don't generally include/overwrite it unless we need some bespoke handling of status for a particular spider. In this case, given I don't think we need need any bespoke handling for status for this agency (unless I'm missing something), you should be able to delete this method and rely on the parent method.

Reply:
Interesting. Yes I just noticed this now after going over the `_get_status` method's implementation that if any word indicating 'cancel' is found from the meeting's `title` or `description` field, status will be processed accordingly. Thanks for pointing this out.